CMakeLists.txt

```diff
@@ -136,6 +155,7 @@ if (${TRITON_PYTORCH_DOCKER_BUILD})
       "libtorch.so"
       "libtorch_cpu.so"
       "libtorch_cuda.so"
+      "libtorch_global_deps.so"
```

Comment:
Is this needed if TRITON_PYTORCH_ENABLE_TORCHTRT is OFF?

Reply:
Well it seems to work without it - on the other hand, that may be just we never encountered the issue. I believe it's safer to always having it linked in.

Reply:
Out of curiosity, what is the issue you encounter if libtorch_global_deps.so is not linked?

Reply:
I am getting link error from libtorchtrt_runtime.so telling that this library is missing.